content/00.front-matter.md

```diff
@@ -52,7 +52,7 @@ Published: {{manubot.date_long}}
   {%- endif %}
   {%- if author.mastodon is defined and author.mastodon is not none and author["mastodon-server"] is defined and author["mastodon-server"] is not none %}
     {%- set has_ids = true %}
-    · ![Mastodon icon](images/mastodon.svg){.inline_icon}
+    · ![Mastodon icon](images/mastodon.svg){.inline_icon width=16 height=16}
```

Comment:
> like for the docx export?

That very well may be. I always forget about docx. If you could do a quick local test (my machine was recently wiped and I can't quickly run Manubot atm)... remove the dimensions next to `.inline_icon` and see if the docx displays properly in Word. If it's smart enough, it could set the dimensions from the inherent SVG dimensions (attached directly in the SVGs).

Regardless though, the html/pdf exports should only need one or the other. If docx needs dimensions next to `.inline_icon`, we can get rid of them in the SVGs themselves.

Reply:
Yes, I confirm that if I remove the dimensions next to `.inline_icon`, then all icons look big and blurry in the docx file (see below), but the HTML/PDF exports work as expected.

![image](https://user-images.githubusercontent.com/172687/224382112-5cb7d6c8-9a47-46d8-bf91-cf9edebd64c9.png)

If I leave the dimensions next to `.inline_icon` but remove them from the SVG, then everything works in all exports and the icons look even better in docx (see below, before/after):

![output](https://user-images.githubusercontent.com/172687/224383798-ac8a1402-fd17-4f94-a1e1-8689db7446fb.png)

Reply:
Thanks for checking and doing this PR Milton. Can you [enable editing of your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) so I can make some final tweaks?

Reply:
@vincerubinetti thank you, just pushed those changes. We should be ready to merge.